site/babel.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (api) =>
+    api.env("test")
+        ? {
+              presets: [
+                  ["@babel/preset-env", { targets: { node: "current" } }],
+                  "@babel/preset-react",
+                  "@babel/preset-typescript",
+              ],
+              plugins: ["@babel/plugin-proposal-optional-chaining"],
```

Comment:
We shouldn't need this plugin anymore with the newer versions of react

Reply:
done